### PR TITLE
More maintenance for 5.10.0

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Review by nuttycom — DISMISSED** [review]
utACK [d8bd0fb](https://github.com/zcash/zcash/pull/6913/commits/d8bd0fb13febb540a0efc7c628e805d398ef2f4c) but it appears that there are clippy lints to be fixed.

**Review by daira — DISMISSED** [review]
This needs a release notes entry saying that `zcash-inspect` has been removed and where it is now. Otherwise utACK.

File: src/rust/src/incremental_merkle_tree.rs [hunk 1]
```diff
@@ -181,6 +184,7 @@ pub fn write_bridge<H: HashSer + Ord, W: Write>(
 /// ordinary circumstances, the checkpoint ID will be the block height at which the checkpoint was
 /// created, but since we don't have any source for this information, we require the caller to
 /// provide it; any unique identifier will do so long as the identifiers are ordered correctly.
+#[allow(clippy::needless_borrows_for_generic_args)]
```
Comment (daira) [inline src/rust/src/incremental_merkle_tree.rs:187]: It doesn't look to me as though there are any needless borrows in this method. Is this a false-positive that should be reported against clippy?

**Review by daira — APPROVED** [review]
utACK

**Comment by str4d** [comment]
[Force-pushed](https://github.com/zcash/zcash/compare/b4c6201f7f5e0ea030d282a241dfa8f129394693..f11acd5e5b07b9bf983fcd0bdd02007d7a784bc8) to fix CI failure.

**Review by nuttycom — APPROVED** [review]
utACK f11acd5e5b07b9bf983fcd0bdd02007d7a784bc8

**Pull request merged**